dist/obs-server.spec

```diff
@@ -276,7 +269,6 @@ Summary:        The Open Build Service -- base configuration files
 Group:          Productivity/Networking/Web/Utilities
 %if 0%{?suse_version}
 Requires(pre):  shadow
-PreReq:         %fillup_prereq
```

Comment:
no, this is still needed for the %fillup_only call,
should probably be replaced by
Requires(pre): %fillup_prereq

Reply:
Done.